src/main/java/org/trips/service_framework/services/CmsService.java

```diff
@@ -100,19 +103,19 @@ public CmsSkuResponse getSkuByCode(String code) {
 
         CmsSkuResponse response = skuSearchHelper(searchSkus);
 
-        if (Objects.isNull(response) || Objects.isNull(response.getData())) {
-            throw new CmsException(String.format("SKU search by code returned null response. Search code: %s", code));
+        if (response == null || response.getData() == null || response.getData().getSearchSkus().isEmpty()) {
```

Comment:
any specific reason to replace Objects.isNull(response) with response == null? @cf-aagam

Reply:
no specific reason, It just mistakenly updated when I updating the logic for this method